app/presenters/api_docs/schema_example.rb

```diff
@@ -0,0 +1,88 @@
+module APIDocs
+  class SchemaExample
+    attr_reader :main_schema
+
+    def initialize(main_schema)
+      @main_schema = main_schema
+    end
+
+    def as_json
+      generate_example_from_schema(main_schema)
+    end
+
+  private
+
+    def generate_example_from_schema(schema_data)
+      properties = {}
+
+      schema_data.properties.each do |key, property|
+        properties[key] = property
+      end
+
+      properties.merge!(get_all_of_hash(schema_data))
+
+      example_data = {}
+
+      properties.each do |property_key, property|
+        example_data[property_key] = generate_example_value_for_property(property)
+      end
+
+      example_data
+    end
+
+    def generate_example_value_for_property(property)
+      if property.example
+        property.example
+      elsif property.type == "object"
+        if property.items
+          generate_example_from_schema(property.items)
+        elsif property.properties
```

Comment:
💥

Reply:
What‽‽ It's ... it's ... beautiful! 😝 

I'm going to blame Apply for this one, again. I'm not sure of the value of refactoring this right at the moment.